tests/jballerina-unit-test/src/test/resources/test-src/expressions/literals/identifierliteral/identifier-literal-success.bal

```diff
@@ -1,130 +1,299 @@
+// Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
If this is a new file shall we change the year?

Reply:
The file was created in 2017 but without license header. I added it.